projects/api-client/src/api/build.service.ts

```diff
@@ -227,12 +344,12 @@ export class BuildService {
 
         // to determine the Content-Type header
         const consumes: string[] = [
+            'application/json'
         ];
 
-        return this.httpClient.put<MainBuild>(`${this.basePath}/build/${encodeURIComponent(String(buildid))}`,
+        return this.httpClient.post<any>(`${this.basePath}/builds/search`,
```

Comment:
Any idea why this is generically typed to `any`? If its generated a speculation might be intresting to see if there is room for upstream improvements. 
If not generated perhaps just remove the typing to any?